src/usr/local/www/vpn_ipsec_mobile.php

```diff
@@ -470,15 +502,30 @@ function login_banner_change() {
 	true
 ))->setHelp('Source');
 
-$section->addInput(new Form_Select(
+$section->addInput(new Form_Checkbox(
 	'group_source',
-	'*Group Authentication',
+	'Group Authentication',
+	'Group Authentication',
 	$pconfig['group_source'],
-	array(
-		'none' => gettext('none'),
-		'system' => gettext('system'),
-	)
-))->setHelp('Source');
+))->setHelp('Authenticate users from the groups that have "User - VPN: IPsec with Dialin" and "WebCfg - All pages" privileges.')
```

Comment:
The code says

```php
if (($priv == 'page-all') || ($priv == 'user-ipsec-xauth-dialin'))
```

but the text says

> that have "User - VPN: IPsec with Dialin" **and** "WebCfg - All pages" privileges

I think that an **and** should be an **or** here, if I understand the intention correctly. In any case I wouldn't want to forced to give `WebCfg - All pages` to my VPN users.

Reply:
Thanks to our wonderful English language both are acceptable. While it's less ambiguous to use "or" in that context, it also doesn't say both are required, thus it can be read as "groups that have A and [groups that have] B".

That said, I am inclined to agree that it should be changed to "or".

Reply:
Change this:
> Authenticate users from the groups that have "User - VPN: IPsec with Dialin" and "WebCfg - All pages" privileges.

To this:
> Authenticate members of groups which have either "User - VPN: IPsec with Dialin" or "WebCfg - All pages" privileges.

Reply:
Ok, all fixed